.github/workflows/docs.yml

```diff
@@ -0,0 +1,49 @@
+name: Docs
+
+on:
+  release:
+    types: 
+      - created
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Checkout docs
+        uses: actions/checkout@v3
+        with:
+          repository: dappnode/DAppNodeDocs
+
+      - name: Set up dappnodesdk
+        run: npm install -g @dappnode/dappnodesdk@${{ github.event.release.tag_name }}
```

Comment:
This is weird, you are getting the sdk from npm instead of from local, changes in branches will not make an effect in gha until merged and release if following this approach.